lib/internal/Magento/Framework/App/ObjectManager/ConfigWriter/Filesystem.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ *
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Framework\App\ObjectManager\ConfigWriter;
+
+use Magento\Framework\App\Filesystem\DirectoryList;
+use Magento\Framework\App\ObjectManager\ConfigWriterInterface;
+
+class Filesystem implements ConfigWriterInterface
```

Comment:
Please add a description to the class or at least inherit it from the interface with `@inheritdoc`

Reply:
I meant this one